Lesson3/HomeWork.py

```diff
@@ -0,0 +1,119 @@
+"""
+HomeWork after lesson 3.
+
+1 - sort list by age and sex fields
+2 - somehow you need to get new list as old list without first
+two elements and last two elements. Print this new list
+3 - in new list calculate total numbers of "female"  and "male"
+and print it as small table.
+
+"""
+
+from colorama import Fore, Style
+
+
+def return_list_slice(sequence: list):
+    """
+    Return a list, excluding the first two and the last two elements.
+
+    Args:
+        sequence (list): The input list.
+
+    Returns:
+        list: The slice of the input list.
+
+    Raises:
+        TypeError: If the input is not a list.
+
+    """
+    if isinstance(sequence, list):
+        return sequence[2:-2]
+    else:
+        raise TypeError('Function parameter must be a list')
+
+
+def count_genders(people_list: list):
+    """
+    Count the number of "female" and "male" in the input list.
+
+    Args:
+        people_list (list): The input list.
+
+    Returns:
+        print: The number of "female" and "male"
+
+    """
+    female, male = 0, 0
+    for person in people_list:
+        if person[-1] == 'female':
+            female += 1
+        elif person[-1] == 'male':
+            male += 1
+    global_width, row_width = 32, 12
+
+    return print('-' * global_width, ' ', '    \n',
```

Comment:
Best practices - > in functions always return certain value . Print should be used in main flow. This will simplify for example debug/unit tests writing for module functional etc.